api/models.py

```diff
@@ -81,29 +81,37 @@ def update(self):
 class User(ModelId):
     """API user model"""
     username: str
-    hashed_password: str
-    active: bool
+    hashed_password: str = Field(description='Hash of the plaintext password')
+    active: bool = Field(default=True,
+                         description='To check if user is active or not')
 
 
 class Revision(BaseModel):
     """Linux kernel Git revision model"""
-    tree: str
-    url: str
-    branch: str
-    commit: str
-    describe: Optional[str] = None
+    tree: str = Field(description='git tree of the revision')
+    url: HttpUrl = Field(description='git URL of the revision')
+    branch: str = Field(description='git branch of the revision')
+    commit: str = Field(description='git commit SHA of the revision')
+    describe: Optional[str] = Field(None, description='git describe of \
+the revision')
```

Comment:
Please break long lines between variables to avoid splitting strings:
```suggestion
    describe: Optional[str] = Field(
        default=None, description='git describe of the revision'
    )
```
Searching the code works better when strings aren't broken, and having code aligned to the left makes it harder to read.

Reply:
Done with the changes.